modules/javafx.controls/src/shims/java/javafx/scene/control/TableColumnShim.java

```diff
@@ -30,5 +30,9 @@ public static void setTableView(TableColumn tc, TableView tv) {
         tc.setTableView(tv);
     }
 
+    // NOTE: These constants are direct copy of TableColumnBase
+    public static final double DEFAULT_WIDTH = 80.0F;
```

Comment:
Can you define these to reference those constants rather than copying them, like this?

```
     public static final double DEFAULT_WIDTH = TableColumnBase.DEFAULT_WIDTH ;
```

If so, you can remove the comments about needing to keep them in sync.